geo_to_hca/utils/entrez_client.py

```diff
@@ -48,13 +47,17 @@ def call_esummary(accession, db='gds'):
 
 def get_entrez_esearch(term, db="sra"):
     throttle()
-    esearch_response = requests.get(url=f'{config.EUTILS_BASE_URL}/esearch.fcgi',
-                     params={
-                         "db": db,
-                         "term": term,
-                         "usehistory": "y",
-                         "format": "json",
-                     })
+    params={
+        "db": db,
+        "usehistory": "y",
+        "format": "json",
+    }
+    if len(term) <= 200:
```

Comment:
Please don't use "magic numbers" in the code.
Define it as a constant.
If the number's source's external documentation, please add a link to the relevant document.

Reply:
The documentation is very cryptic, literally says `more than several hundred characters long` for eSearch, and `if more than about 200 UIDs are to be provided, the request should be made using the HTTP POST method`. That's why I chose 200, the `terms` field can probably take more characters but I'd need to make tests to see how many hundred.

The documentation is here https://www.ncbi.nlm.nih.gov/books/NBK25499/#chapter4.ESearch for Esearch and https://www.ncbi.nlm.nih.gov/books/NBK25499/#_chapter4_EFetch_ for EFetch

Reply:
I will add it as a constant, though!